### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: html/donate/de.html [hunk 1]
```diff
@@ -166,7 +166,7 @@ <h3>Helfen Sie uns, den Etat von Open Food Facts 2020 zu finanzieren!</h3>
           Infolgedessen haben wir das Projekt mit einem Schnürsenkel gestartet, und wir halten unser Budget immer noch knapp. Von 2012 bis 2017 waren wir alle Freiwillige und unser Jahresbudget betrug weniger als 1.000 Euro! Das Projekt ist jedoch stark gewachsen, und wir brauchen jetzt etwas mehr, um unsere Mission weiterzuführen.
         </p>
         <p>
-          Wir benötigen Ihre Hilfe, um Open Food Facts weiter zu betreiben und zu wachsen. Bitte denken Sie darüber nach, eine Spende zu leisten, um Open Food Facts zu schützen und zu erhalten.
+          https://donorbox.org/embed/your-donation-to-open-food-facts
```
Comment (stephanegigandet) [inline html/donate/de.html:169]: Problem?

File: html/donate/fr.html [hunk 2]
```diff
@@ -172,7 +172,7 @@ <h3>Aidez-nous à financer le budget Open Food Facts 2020 !</h3>
           grandi, et nous avons maintenant besoin de plus de ressources pour continuer notre mission.
         </p>
         <p>
-          Nous avons besoin de votre aide pour continuer à gérer et faire grandir Open Food Facts. Envisagez de faire un don pour protéger et soutenir Open Food Facts.
+          Open Food Facts
```
Comment (stephanegigandet) [inline html/donate/fr.html:175]: Problem? also "Envisagez de" sounds weird.

**Comment by teolemon** [comment]
Borked Crowdin manipulation. It added 9000 translations in many languages out of the blue, I reverted on Crowdin, and it was a bad idea.
I'm fixing this

File: po/openproductsfacts/tr.po [hunk 3]
```diff
@@ -42,7 +42,7 @@ msgstr "openproductsfacts-logo-en-356x300.png"
 
 msgctxt "site_name"
 msgid "Open Products Facts"
-msgstr "Open Products Facts"
+msgstr "Açık Ürün Gerçekleri"
```
Comment (teolemon) [inline po/openproductsfacts/tr.po:45]:
```suggestion
msgstr "Open Products Facts"
```

File: po/openproductsfacts/ro.po [hunk 4]
```diff
@@ -42,7 +42,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Products Facts"
-msgstr "Open Products Facts"
+msgstr "Deschideți informații despre produse"
```
Comment (teolemon) [inline po/openproductsfacts/ro.po:45]:
```suggestion
msgstr "Open Products Facts"
```

File: lang/ar/texts/mobile-app-open-food-facts.html [hunk 5]
```diff
@@ -2,7 +2,7 @@ <h1>Install the Open Food Facts mobile app</h1>
 
 <p>To scan food products, get their Nutri-Score, their NOVA group to know if it's an ultra-processed food, and to decypher food additives, install the free Open Food Facts app!</p>
 
-		<a href="https://apps.apple.com/ar/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
+		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
```
Comment (teolemon) [inline lang/ar/texts/mobile-app-open-food-facts.html:5]:
```suggestion
		<a href="https://apps.apple.com/ar/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
```

File: lang/ar/texts/who-we-are.html [hunk 6]
```diff
@@ -17,7 +17,7 @@ <h2>المساهمون</h2>
 
 &rarr; <a href="https://twitter.com/openfoodfacts">Twitter</a><br/>
 &rarr; <a href="https://www.facebook.com/OpenFoodFacts">Facebook page</a>
-&rarr; <a href="https://www.instagram.com/Open.Food.Facts">Instagram page</a>
+&rarr; <a href="https://www.instagram.com/Open. Food. Facts">Instagram page</a>
```
Comment (teolemon) [inline lang/ar/texts/who-we-are.html:20]:
```suggestion
&rarr; <a href="https://www.instagram.com/Open.Food.Facts">Instagram page</a>

```

File: po/openproductsfacts/hu.po [hunk 7]
```diff
@@ -110,7 +110,7 @@ msgstr ""
 
 msgctxt "footer_wiki_link"
 msgid "https://wiki.openfoodfacts.org"
-msgstr ""
+msgstr "https://wiki.openfoodfacts.org&hl=hu"
```
Comment (teolemon) [inline po/openproductsfacts/hu.po:113]:
```suggestion
msgstr "https://wiki.openfoodfacts.org"
```

File: po/openproductsfacts/ha.po [hunk 8]
```diff
@@ -42,7 +42,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Products Facts"
-msgstr "Open Products Facts"
+msgstr "Bude samfura na Gaskiya"
```
Comment (teolemon) [inline po/openproductsfacts/ha.po:45]:
```suggestion
msgstr "Open Products Facts"
```

File: po/openpetfoodfacts/ro.po [hunk 9]
```diff
@@ -28,7 +28,7 @@ msgstr "openfoodfacts-logo-en-356x300.png"
 # Do not translate Open Pet Food Facts
 msgctxt "site_name"
 msgid "Open Pet Food Facts"
-msgstr "Open Pet Food Facts"
+msgstr "Deschideți informații despre alimentele pentru animalele de companie"
```
Comment (teolemon) [inline po/openpetfoodfacts/ro.po:31]:
```suggestion
msgstr "Open Pet Food Facts"
```

File: po/openpetfoodfacts/hu.po [hunk 10]
```diff
@@ -126,7 +126,7 @@ msgstr ""
 
 msgctxt "footer_wiki_link"
 msgid "https://wiki.openfoodfacts.org"
-msgstr ""
+msgstr "https://wiki.openfoodfacts.org&hl=hu"
```
Comment (teolemon) [inline po/openpetfoodfacts/hu.po:129]:
```suggestion
msgstr "https://wiki.openfoodfacts.org"
```

File: lang/ar/texts/nutriscore.html [hunk 11]
```diff
@@ -58,7 +58,7 @@ <h3>Install the Open Food Facts mobile app</h3>
 
 <p>To scan food products, get their Nutri-Score nutritional grade, the NOVA group for ultra-processed foods, allergens alerts and to decypher food additives, install the free Open Food Facts app!</p>
 
-		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40"></a>
+		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_US.svg" alt="Available on the App Store" width="120" height="40"></a>
```
Comment (teolemon) [inline lang/ar/texts/nutriscore.html:61]:
```suggestion
		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40"></a>
```

**Pull request closed without merging**